hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/token/TokenCreateSpecs.java

```diff
@@ -868,7 +894,13 @@ public HapiSpec creationValidatesSymbol() {
                         recordSystemProperty("tokens.maxSymbolUtf8Bytes", Integer::parseInt, maxUtf8Bytes::set))
                 .when()
                 .then(
-                        tokenCreate("missingSymbol").symbol("").hasPrecheck(MISSING_TOKEN_SYMBOL),
+                        tokenCreate("missingSymbol")
+                                .symbol("")
+                                // Either fail pre-check with MISSING_TOKEN_SYMBOL (mono)
+                                // or post-consensus (modular). When we only have modular code left,
+                                // we can just check hasKnownStatus.
+                                .hasPrecheckFrom(OK, MISSING_TOKEN_SYMBOL)
+                                .hasKnownStatus(MISSING_TOKEN_SYMBOL),
```

Comment:
Instead of this you can add `MISSING_TOKEN_SYMBOL` to this property `spec.streamlinedIngestChecks`. This will avoid modifying tests

Reply:
Thank you! 🎉  I didn't know about this property.